tests/integration/test_dataflow_integration.py

```diff
@@ -72,6 +78,15 @@ def test_dataflow_integration():
 
         # okay, time to start checking if the job is done
         show_job = f"gcloud dataflow jobs show {job_id} --format=json".split()
+        show_job_errors = [
```

Comment:
reviewer note: let's use this command to dump logs when job fails so we don't have to use GCP console